### ISSUE TITLE
Add utteranc.es?

### ISSUE THREAD

**Issue by PoignardAzur** [matklad/matklad.github.io#161]
[utteranc.es](https://utteranc.es/) is a quick and dirty way to add comments to your github blog.

Sometimes you publish an article and then you say something wrong and I want to write about how wrong you are and stuff, but since you haven't linked a reddit thread, I can't!

Having utterances would fix that problem! (It would also let people say nice things.)

**Comment by matklad** [comment]
I love discussion, but I don't love to be responsible for moderation :-) I do love email though! e-mail is a-w-e-s-o-m-e! Added a more prominent link and eager to be disagreed with :D

**Comment by matklad** [comment]
https://github.com/matklad/matklad.github.io/commit/aee465e0ea9e65c333ab8789152c4031d33b1b54

**Comment by matklad** [comment]
Yup, looks like adding email link was a very good decision, people do reach out to me with the provided `+blog` alias, thanks for the poke here @PoignardAzur!

**Comment by PoignardAzur** [comment]
But then how will other people see my brilliant comments about how wrong you are?

Fair point about moderation, though.

**Comment by matklad** [comment]
I actually semi-quit redding&lobste.rs because I noticed that "people see my brilliant" comments is one of my motivations to engage with those platforms a bit more than what I'd rationally prefer. Plus, aesthetically, that was for me "see my" first and foremost, rather "people learn brilliant things". Which again is a totally understandable circuit it my brain, but not the one I'd love tickling, if I could help that.

So I figured that:

* for actually brilliant things which I want to get out there in the world, I'd just go and write a blog post (and, to save time, won't really edit it, and release with typos and such. Basically, a HN comment, but detached from someone else's spotlight)
* for dialogs, which I also value, I actually prefer email, as that completely removes the signalling aspect, and ensures that the two communicate to share thoughts with each other, rather than to show bystanders how smarter than the other they are.

**Comment by PoignardAzur** [comment]
Ah, yeah. I suppose that's a good mindset to adopt.
